### DISCUSSION TITLE
[Core Protocol] Ecotone fee docs are incorrect and inconsistent

### DISCUSSION THREAD

**Discussion by deanpress** [ethereum-optimism/developers#429] [Core Protocol] [answered]
### Did you check the documentation?

- [X] Yes

### Did you read the specs?

- [X] Yes
- [ ] No

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

The ecotone section on [fees](https://docs.optimism.io/builders/app-developers/transactions/fees) explains the following:

> At the exact point of the Ecotone upgrade, the dynamic overhead parameter value is used to initialize the Ecotone base fee scalar, and **blob base fee is set to 0.** The overhead parameter from the previous function becomes ignored.

On the final line, it says:

> Recall that base_fee_scalar is set to dynamic_overhead and **blob_base_fee_scalar is 0** 

Meanwhile, looking at GasOracle calls in a [transaction stack trace](https://dashboard.tenderly.co/tx/base/0xe07d92b75887eae9836b5a54bc29e3766ffdf75d1876743648abf1e7bf3dfed8), none of the values returned are 0 (even though the baseFee view function in [GasOracle on basescan](https://basescan.org/address/0x420000000000000000000000000000000000000F#readProxyContract#F2) returns 0).

Following the formula in the Optimism docs using GasOracle will result in an L1 fee calculation that's much (>10x) higher than the actual L1 fee (e.g. `800 * (16 * 1101 * 5931493) + (659851 * 2)`).

Please provide an accurate description of the calculation in the docs. I believe a Solidity example using GasOracle will also help greatly.

**Comment by hash1go** [comment]
Hello. Are you asking about Base rather than OP Mainnet?

**Reply by hash1go** [reply, marked as answer]
@deanpress 
First, the values you referred are no longer in use.

> (even though the baseFee view function in [GasOracle on basescan](https://basescan.org/address/0x420000000000000000000000000000000000000F#readProxyContract#F2) returns 0).

Please refer to [this](https://basescan.org/address/0x420000000000000000000000000000000000000F#readProxyContract#F11).
[That document](https://docs.optimism.io/builders/app-developers/transactions/fees) does not reflect the actual method names of the contract. Therefore, I think it would be best to refer to [this specification](https://specs.optimism.io/protocol/exec-engine.html#ecotone-l1-cost-fee-changes-eip-4844-da).

Additionally, the `base_fee_scalar` and `blob_base_fee_scalar` have the hardcoded value of `6` applied as [decimals](https://basescan.org/address/0x420000000000000000000000000000000000000F#readProxyContract#F1). Therefore, you should divide by `10e6`. For more details, please refer to [this specification](https://specs.optimism.io/protocol/predeploys.html#gaspriceoracle).

Based on this, the L1fee should output the actual value.
Upon actual calculation, your L1 fee can be 0.00013 USD.
